tests/test_keyboard_shortcuts.py

```diff
@@ -7,6 +7,53 @@
 from pages.treeherder import TreeherderPage
 
 
+def test_close_open_panels(base_url, selenium):
+    """Open Treeherder, verify shortcut: 'Esc' closes filter and job panel.
+    Open Treeherder page, open Filters panel, select random job, close all
+    panels using 'esc' button, verify if all panels are closed.
+    """
+    page = TreeherderPage(selenium, base_url).open()
+
+    page.click_on_filters_panel()
+    page.select_random_job()
+
+    assert page.filter_panel_is_open
+    assert page.info_panel_is_open
```

Comment:
I think I would put job_details within info_panel, but let's not increase the scope of this patch. Maybe add a TODO comment in JobDetails to move it into the InfoPanel region, as it would require some tests to be modified.